### PR TITLE
SPCTR-3220: Improvement:  Info-Box prefix heading tags

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Added heading tags option for prefix of Info-box as per ZIPWP request.
https://brainstormforce.atlassian.net/browse/SPCTR-3220

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +220,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Improvement: Image - Added a new `Custom Size` feature, which will maintain the dimensions set when the image is replaced!
 * Improvement: Popup Builder - Moved page-level settings into block-level panels.
 * Improvement: Testimonial - Added a vertical alignment option for equal-height content.
+* Improvement: Info Box - Added heading tags settings for prefix.
```
Comment (bsf-zanev) [inline README.md:223]: Move this entry below the Image Improvements.

```suggestion
* Improvement: Info Box - Added heading tag settings for prefix.
```

File: src/blocks/info-box/attributes.js
Comment (bsf-zanev) [inline src/blocks/info-box/attributes.js:1]: The front-end attribute default has not been set - What will be the result on the front-end when no tag is selected...?

**Review by bsf-zanev — COMMENTED** [review]
Just had one doubt, and the changelog entry needs to be reordered with a minor change.

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md:223]: Done.

File: src/blocks/info-box/attributes.js
Comment (rajmohanreddy774) [reply src/blocks/info-box/attributes.js:1]: That is handled in save.js by using attributes.prefixHeadingTag.

File: README.md (hunk 1 above)
Comment (bsf-zanev) [reply README.md:223]: Could you rephrase as suggested above after pulling in the latest dev?

Also move it to a new section, since we've just released 2.7.4.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by adi3890 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adi3890) [reply README.md]: @bsf-zanev @rajmohanreddy774 please resolve this commetn if done.

**Review by adi3890 — APPROVED** [review]
